server/bleep/src/agent.rs

```diff
@@ -432,7 +432,7 @@ impl Agent {
             target: Some(query),
             repos: repos
                 .iter()
-                .map(RepoRef::to_string)
+                .map(RepoRef::display_name)
```

Comment:
```suggestion
                .map(RepoRef::indexed_name)
```

We can use `indexed_name` and remove this block: https://github.com/BloopAI/bloop/blob/projects-fix-local-code-search/server/bleep/src/semantic.rs#L811-L817